src/plugins/dashboard/server/content_management/v3/cm_services.ts

```diff
@@ -437,6 +437,7 @@ export const dashboardSearchOptionsSchema = schema.maybe(
     {
       onlyTitle: schema.maybe(schema.boolean()),
       fields: schema.maybe(schema.arrayOf(schema.string())),
+      includeReferences: schema.maybe(schema.arrayOf(schema.string())),
```

Comment:
Should we limit the type of reference we allow instead of any string? e.g for now only `tags` ?